ragatouille/models/colbert.py

```diff
@@ -161,64 +154,56 @@ def add_to_index(
                     self._get_collection_files_from_disk(
                         str(Path(index_root) / self.index_name)
                     )
+            else: self.collection = []
```

Comment:
Ah, this is why I was unable to add a single document as per my previous comment: this line wipes out the previous collection, so we try to rebuild an index consisting solely of the one new document. I don't understand what the intended purpose of this line is supposed to be? It seems like we'd never add_to_index() to delete the pre-existing collection.

Reply:
You're right -- I think this line is there as a remnant of an earlier bug that got reproduced as part of this transition. It won't make it to the new version release fixing these issues!

Reply:
@jlscheerer I understand the overall logic, but I think this'd be my only objection (on a nitpicky level) before merging: is there a specific reason for emptying `self.collection`? 

f it's empty and gets loaded from the `collection.json`, everything's fine, but we'll run into issues here where people build an index, so `self.collection` is set, but want to add to it while keeping the collection 🤔

Reply:
Good catch @ksadov and @bclavie. Didn't notice this as part of my testing. Removing.